<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Aracne</class>
 <widget class="QMainWindow" name="Aracne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aracne</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="spiderButton">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>670</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Spider</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dumpButton">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>670</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Dump</string>
    </property>
   </widget>
   <widget class="QListWidget" name="requestsWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>621</width>
      <height>621</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1250</width>
      <height>1250</height>
     </size>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="repliesWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>601</width>
      <height>620</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1250</width>
      <height>620</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="requestButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Request</string>
    </property>
   </widget>
   <widget class="QPushButton" name="replyButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Reply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startServerButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>670</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopServerButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>670</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Server</string>
    </property>
   </widget>
   <widget class="QLabel" name="serverLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>670</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1275</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
